superset-frontend/src/explore/components/DatasourcePanel.tsx

```diff
@@ -175,8 +175,12 @@ const DataSourcePanel = ({
       return;
     }
     setList({
-      columns: matchSorter(columns, value, { keys: ['column_name'] }),
-      metrics: matchSorter(metrics, value, { keys: ['metric_name'] }),
+      columns: matchSorter(columns, value, {
+        keys: ['column_name', 'expression', 'description', 'verbose_name'],
+      }),
+      metrics: matchSorter(metrics, value, {
+        keys: ['metric_name', 'expression', 'description', 'verbose_name'],
```

Comment:
description and verbose_name should probably rank before expression.

Reply:
got it